tools/theme/test/themes.test.ts

```diff
@@ -165,7 +165,7 @@ describe('Setting attributes', () => {
         expect(el.hasAttribute('scale')).to.be.false;
 
         if (el.shadowRoot.adoptedStyleSheets) {
-            expect(el.shadowRoot.adoptedStyleSheets.length).to.equal(2);
+            expect(el.shadowRoot.adoptedStyleSheets.length).to.equal(3);
```

Comment:
Why are we adding an additional stylesheet?

Reply:
Nevermind. This was something I was testing, it accidentally got into the PR